sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -769,7 +769,7 @@ inlineTable
     ;
 
 functionTable
-    : funcName=functionName LEFT_PAREN (expression (COMMA expression)*)? RIGHT_PAREN tableAlias
+    : funcName=functionName LEFT_PAREN (functionArgument (COMMA functionArgument)*)? RIGHT_PAREN tableAlias
```

Comment:
here in the `.g4` file, we only change the syntax for table function calls, but the PR title and description mention general named argument support. Can we either update the PR description or else update the `functionCall` rule to replace `argument+=expression` with `argument+=functionArgument`? The latter would be nice since we can make all the parser changes in the same PR.